README.md

```diff
@@ -3,41 +3,43 @@ pytest-tinybird
 
 A pytest plugin to report test results to tinybird
 
+[![PyPI version](https://badge.fury.io/py/pytest-tinybird.svg)](https://badge.fury.io/py/pytest-tinybird)
 ![Passed](https://github.com/jlmadurga/pytest-tinybird/actions/workflows/main.yml/badge.svg)
 ![Top test passed](https://img.shields.io/endpoint?url=https://api.tinybird.co/v0/pipes/top_test_passed.ndjson?token=p.eyJ1IjogIjNhZjhlMTBhLTM2MjEtNDQ3OC04MWJmLTE5MDQ5N2UwNjBjYiIsICJpZCI6ICJkNDNmZGQ2Ni03NzY1LTQzZGYtYjEyNS0wYzNjYWJiMDgxZjUifQ.yWypEczMfJlgkjNt29pCf45XaxE1dMOr-oznll5tjpY)
 
 
+
 Requirements
 ------------
 
 - Python >=3.8
 - pytest 3.8 or newer (previous versions might be compatible)
+- Tinybird account and a token with append permissions
```

Comment:
We may have an opportunity here to add a link with some instructions on how to do it, just to make the process smoother. Do you know whether we have something suitable to add here?